google_scraper.py

```diff
@@ -55,22 +55,26 @@ def search(url):
             element.send_keys(Keys.PAGE_DOWN)
             time.sleep(0.3)  # bot id protection
 
-    print("[%] Reached end of Page.")
+    print("==> Reached end of page")
 
     time.sleep(1)
     # Get page source and close the browser
+    print("==> Saving page source...")
     source = browser.page_source
-    f = open('dataset/logs/google/source.html', 'w+')
-    f.write(source)
+    with open('dataset/logs/google/source.html', 'w+', encoding='utf-8', errors='replace') as f:
+        f.write(source)
 
+    print("==> Page source saved")
+	
     browser.close()
-    print("[%] Closed Browser.")
+    print("==> Closed browser window")
+    print("\n===============================================\n")
 
     return source
 
 
 def error(link):
-    print("[!] Skipping {}. Can't download or no metadata.\n".format(link))
+    print("==> Skipped. Can't download or no metadata.\n")
```

Comment:
I would want to keep the link for reference purposes. Since this is logged as an error (`[!]`) users can iterate the links to manually download the images.